src/pod.ts

```diff
@@ -165,6 +167,39 @@ export class Pod {
     return this.cache.docs[key];
   }
 
+  /**
+   * Lists documents using glob patterns, as outlined by the [`glob`
+   * module](https://github.com/isaacs/node-glob#glob-primer).
+   *
+   * Note the following behavior:
+   * - Files prefixed with `_` are ignored.
+   * - Only files with supported doc extensions are returned.
+   *
+   * Various techniques can be used to list docs depending on your needs:
+   *
+   * ```
+   * // All docs within the "pages" collection:
+   * pod.docs('/content/pages/**')
+   *
+   * // Only Markdown docs within the "pages" collection:
+   * pod.docs('/content/pages/*.md')
```

Comment:
Would this be better as `/content/pages/**/*.md` to match `.md` in all directories within the collection?